react/components/ProductQuantitySelector/README.md

```diff
@@ -0,0 +1,128 @@
+# VTEX Product Quantity Selector
+
+## Description
+
+The VTEX Product Quantity Selector allows the user to add to cart the specified amount of the displayed product
+
+## Table of Contents
+
+- [Usage](#usage)
+  - [Blocks API](#blocks-api)
+    - [Configuration](#configuration)
+  - [Styles API](#styles-api)
+    - [CSS Namespaces](#css-namespaces)
+
+## Usage
+
+This app uses our store builder with the blocks architecture. To know more about Store Builder [click here.](https://help.vtex.com/en/tutorial/understanding-storebuilder-and-stylesbuilder#structuring-and-configuring-our-store-with-object-object)
+
+To use this app or override the default CSS you need import it in your dependencies on `manifest.json` file.
+
+```json
+  "dependencies": {
+    "vtex.product-details": "1.x"
```

Comment:
copy paste? :grimacing:

Reply:
no time brother

(a lot of documentation needs to be updated, this will be done after vtex day)